Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "MIT OR Apache-2.0"
 repository = "https://github.com/amPerl/egui-phosphor"
 
 [dependencies]
-egui = { version = "0.26", default-features = false }
+egui = { version = "0.27", default-features = false }
```

Comment:
Just as a note, because not many people seem to be using this feature, you can specify a version range, like `">=0.26.0,<=0.27.0"`, so it both supports 0.26 and 0.27

Reply:
awesome, did not know that, thanks!